src/main/java/com/powsybl/openloadflow/sa/OpenSecurityAnalysis.java

```diff
@@ -269,7 +307,11 @@ private SecurityAnalysisResult runSimulations(LfNetwork network, List<Propagated
             }
 
             LimitViolationsResult preContingencyResult = new LimitViolationsResult(preContingencyComputationOk, new ArrayList<>(preContingencyLimitViolations.values()));
-            return new SecurityAnalysisResult(preContingencyResult, postContingencyResults);
+            return new SecurityAnalysisResult(preContingencyResult, postContingencyResults,
```

Comment:
Not a must-do, but maybe it would be easier to do building of the maps inside the constructor, so that we do not have to do it ourselves here